src/binutil/setup.rs

```diff
@@ -35,26 +36,21 @@ pub fn initial_logger(config: &TiKvConfig) {
             .expect("config.log_rotation_timespan is an invalid duration.");
 
     // Collects following targets.
-    let mut enabled_targets = vec![
-        "tikv::".to_owned(),
-        "raft::".to_owned(),
-        "tests::".to_owned(),
-        "benches::".to_owned(),
-        "integrations::".to_owned(),
-        "failpoints::".to_owned(),
-        // Collects logs for test components.
-        "test_".to_owned(),
-    ];
+    let mut enabled_targets = HashSet::new();
```

Comment:
I prefer to simply remove this filter. @overvenus PTAL

Reply:
This filter was added for the purpose of disabling debug log of other dependencies. Though I'd like to remove it too.🤣

Reply:
Maybe we need to do blacklist instead? Looks like debug logs from other dependencies are very rare, limited to only a few crates, whose number is fewer than our own components 🤣

Reply:
good idea, maybe set empty for the default blacklist and set the env variable manually when necessary.